src/main/java/com/solvd/ecommerce/ConfigReader.java

```diff
@@ -0,0 +1,22 @@
+package com.solvd.ecommerce;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.util.Properties;
+
+public class ConfigReader {
+
+    private static final Properties properties = new Properties();
+
+    static {
+        try {
+            properties.load(new FileInputStream("src/main/resources/config.properties"));
+        } catch (IOException e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    public static String getValue(String key) {
+        return properties.getProperty(key);
+    }
+}
```

Comment:
Better to move this class on additional package like "utils", and create package for pages "page".